odoorpc/odoo.py

```diff
@@ -65,6 +65,7 @@ def __init__(
         timeout=120,
         version=None,
         opener=None,
+        **kwargs,
```

Comment:
Can't you just add the autoretry arg?

```suggestion
        autoretry=False,
```